include/aie/Dialect/AIE/Transforms/AIEPasses.td

```diff
@@ -27,8 +27,8 @@ def AIEAssignBufferAddresses : Pass<"aie-assign-buffer-addresses", "DeviceOp"> {
   let constructor = "xilinx::AIE::createAIEAssignBufferAddressesPass()";
 
   let options = [
-    Option<"clBasicAlloc", "basic-alloc", "bool", /*default=*/"false",
-            "Flag to enable the basic sequential allocation scheme (not bank-aware).">
+    Option<"clAllocScheme", "alloc-scheme", "std::string", /*default=*/"",
+           "Choose allocation scheme; possibilities: basic-sequential, bank-aware. By default, basic-sequential is selected for the Tile's memory and bank-aware is selected for the rest.">,
   ];
```

Comment:
I think this should be something along the lines of: By default, the compiler will try to use the bank-aware scheme first and, if it fails, will fallback to the basic-sequential scheme.